datafusion/physical-plan/src/aggregates/topk/heap.rs

```diff
@@ -462,7 +486,7 @@ mod tests {
         let mut heap = TopKHeap::new(10, false);
         heap.append_or_replace(1, 1, &mut map);
 
-        let actual = heap.tree_print();
+        let actual = format!("{heap}");
```

Comment:
Another way to write this is `heap.to_string()`, which might be more "idomatic"

```suggestion
        let actual = heap.to_string()
```

Reply:
sure, I'll fix that